lib/freeclimb/utils/signature_information.rb

```diff
@@ -0,0 +1,44 @@
+require 'date'
+require 'openssl'
+require 'cgi'
```

Comment:
what is this dependency for?

Reply:
It's not needed, will remove in latest commit